docs/production/authentication-methods.md

```diff
@@ -284,6 +284,34 @@ details.
 
 [upstream-ldap-groups]: https://django-auth-ldap.readthedocs.io/en/latest/groups.html#limiting-access
 
+### Restricting LDAP user access to specific organizations
+
+If you're hosting multiple Zulip organizations, you can restrict which
+users have access to which organizations.
+This is done by setting `org_membership` in `AUTH_LDAP_USER_ATTR_MAP` to the name of
+the LDAP attribute which will contain a list of  subdomains that the
+user should be allowed to access.
+
+For the root subdomain, `www` in the list will work, or any other of
+`settings.ROOT_SUBDOMAIN_ALIASES`.
+
+For example, with `org_membership` set to `department`, a user with
```

Comment:
How did we pick the `org_membership` field name?  That's the one remaining hangup I have about the non-documentation parts of this commit -- whether we're choosing the right name for this interface.

Reply:
I think @mateuszmandera set the name.
You mentioned it in https://github.com/zulip/zulip/pull/15659#issuecomment-696525019

As far as I understand we use this name to be consistent with the SAML options https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#restricting-access-to-specific-organizations

Reply:
Yeah, my motivation was using the same name as in saml